### PR TITLE
🎨 IMPROVE: Changing uncontrolled input to be controlled warning in ac…

### PR DESCRIPTION
**Pull request by Humed-Muhammad**
…counts form #198

### Pull Request Checklist: 

- [ ] Please make sure all jobs pass before requesting a review. 
- [ ] Assign yourself to the PR
- [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
- [ ] Label it as "needs-review"
- [ ] Have you lint your code locally before submission?
- [ ] Did you write tests where appropiate?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -58,4 +58,5 @@ settings.json
 
 GoSDK/tester/
 
-.gitconfig
\ No newline at end of file
+.gitconfig
+.env
```
Comment (simlarsen) [inline .gitignore:62]: Please revert these changes.

**Review by simlarsen — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Humed-Muhammad) [reply .gitignore:62]: Ok

Comment (Humed-Muhammad) [reply .gitignore]: Good to go, I revert it but to be clear the .gitconfig was already there, I only added the .env

Comment (simlarsen) [reply .gitignore]: Yes

**Review by simlarsen — APPROVED** [review]
